services/ses/index.js

```diff
@@ -0,0 +1,60 @@
+
+function assertPresent(thing, message) {
+  if (!thing) { throw new Error(message); }
+  return thing;
+}
+
+function assertEmail(email) {
+  ['emailTo', 'message', 'contact']
+    .forEach(field => {
+      assertPresent(email[field], `Missing ${field}`);
+    });
+  return email;
+}
+
+function constructEmail(emailData) {
+  const messageLabel = 'This email was sent through the contact us form on red-badger.com:';
+  const contactLabel = 'Contact details:';
+
+  const formattedHTMLMessage = `<p><strong>${messageLabel}</strong></p><p>${emailData.message}</p><p><strong>${contactLabel}</strong></p><p>${emailData.contact}</p>`;
+
+  const formattedTxtMessage = `${messageLabel}\n\n ${emailData.message}\n\n ${contactLabel}\n\n ${emailData.contact}`;
```

Comment:
Perhaps we can split these strings into multiple lines, especially since they have newlines in them. Would make it much easier to read.